client/vue.config.js

```diff
@@ -2,12 +2,14 @@ module.exports = {
   transpileDependencies: [
     'vuetify',
   ],
-  devServer: {
-    proxy: {
-      '^/api': {
-        target: 'http://localhost:3000',
-        changeOrigin: true
-      },
+  configureWebpack: {
+    devServer: {
+      proxy: {
+        '/api': {
+          target: 'http://localhost:3000',
+          changeOrigin: true
+        },
+      }
```

Comment:
Ano difference nito? We're not pretty sure how this proxy thing works. Na try mo na ba?

Reply:
Para di na mag add ng localhost:3000 to every request may proxy tayo na '/api' which denotes na if nag /api ka, deretso na sa http://localhost:3000/api yung request. Gagana lang to after mag configure ng webpack which is always the case if irurun mo na yung project.

Reply:
i see. 

so /api/users --> localhost:3000/users, right?